exonum/src/storage/values.rs

```diff
@@ -114,11 +114,11 @@ impl StorageValue for bool {
 }
 
 impl StorageValue for u8 {
-    fn into_bytes(self) -> Vec<u8> {
+    fn into_bytes(self) -> Vec<Self> {
```

Comment:
How do you think, is it make sense to use Self? Or restore u8? The same in other places with integral types.

Reply:
When we're using default types I stand for type name instead of `Self`. 
`Self` looks like something that belongs to us, and I won't expect it to be `Vec`, `u8`, `Result` or something similar.

But that's my opinion and other comrades may disagree though.

Reply:
For me it is somewhat strange to see thing like `Vec<Self>` or `HashMap<Self, i32>`. `Result<Self, _>` is OK though.

Reply:
Additionally, in this situations `Vec<u8>` doesn't mean `Vec<Self>` at all, it is `Vec<byte>`.

Reply:
https://github.com/rust-lang-nursery/rust-clippy/issues/2894